include/swift/AST/DiagnosticsParse.def

```diff
@@ -1316,6 +1316,20 @@ ERROR(attr_availability_expected_equal,none,
 ERROR(attr_availability_expected_version,none,
       "expected version number in '%0' attribute", (StringRef))
 
+WARNING(attr_availability_swift_platform_expected_option,none,
+      "expected '%0' option with a version number for swift platform, "
```

Comment:
Nitpick: I suggest using this as "platform 'swift'" (like "platform '*'"). It seems a little clearer to me.

After seeing it in context, I also think it might just be better to simplify to something like "expected 'introduced', 'deprecated', or 'obsoleted' in '%0' attribute for platform 'swift'". It does mean we have to keep it up to date, but it's less circumlocution.